pages/api/locations/read.ts

```diff
@@ -12,15 +12,18 @@ export default async function handler(
       if (typeof req.body === 'string') {
         req.body = JSON.parse(req.body)
       }
-      const { id: I } = req.body.data
-
+      const { id: I } = req.body
```

Comment:
currently we're not passing in any additional information in the body so adding a data field doesn't provide any functional difference. Although this could be helpful in the future if we want to pass other information in the body but not the data object. I'm not sure what that use case would be though